src/mole_C++/utils.cpp

```diff
@@ -81,6 +108,14 @@ sp_mat Utils::spkron(const sp_mat &A, const sp_mat &B) {
   return result;
 }
 
+/**
+ * @brief An in place oepration for two matrices by rows
```

Comment:
nit: typo.

Reply:
fixed in #[commit o295c8f](https://github.com/csrc-sdsu/mole/pull/11/commits/0295c8fdbeebed3a51260cb1276d185a8270844d)